src/config/wmodules-vuln-detector.c

```diff
@@ -416,6 +417,8 @@ int Read_Vuln(const OS_XML *xml, xml_node **nodes, void *d1, char d2) {
                 merror("Invalid content for tag '%s' at module '%s'", XML_RUN_ON_START, WM_VULNDETECTOR_CONTEXT.name);
                 return OS_INVALID;
             }
+        } else if (!strcmp(nodes[i]->element, XML_IGNORE_TIME)){
+            mwarn("Detected deprecated tag <%s> at module '%s'.", nodes[i]->element, WM_VULNDETECTOR_CONTEXT.name);
```

Comment:
As per the discussion that we had, change this log for something like:

```
The <%s> tag at module '%s' is deprecated for version newer than 4.3.
```

Reply:
Done!